deploy/acm-policies/rosa-console-branding.Policy.yaml

```diff
@@ -24,11 +24,19 @@ spec:
                     - complianceType: musthave
                       objectDefinition:
                         apiVersion: operator.openshift.io/v1
-                        applyMode: AlwaysApply
                         kind: Console
-                        name: cluster
-                        patch: '{"spec":{"managementState":"Managed","route":null,"customization":{"brand":null,"documentationBaseURL":"https://docs.openshift.com/rosa/","customProductName":"Red Hat OpenShift Service on AWS","customLogoFile":{"name":"rosa-brand-logo","key":"rosa-brand-logo.svg"}}}}'
-                        patchType: merge
+                        spec:
```

Comment:
Is this part manually modified after 'make' auto generated the policy? If so i'm afraid the fix would not last, since every time someone runs 'make' in this repo, everything is regenerated again and this fix would be overwritten.
one solution would be that we have a script to generate policies for the 'patch'.  https://issues.redhat.com/browse/OSD-15086